settings.gradle.kts

```
@@ -367,6 +367,7 @@ include(":instrumentation:spring:spring-integration-4.1:library")
 include(":instrumentation:spring:spring-integration-4.1:testing")
 include(":instrumentation:spring:spring-kafka-2.7:javaagent")
 include(":instrumentation:spring:spring-rabbit-1.0:javaagent")
+include(":instrumentation:spring:spring-rmi:javaagent")
```

the convention here is to add the min version that the instrumentation supports (the reason the rmi instrumentation has none is because it's supported on all Java versions that are supported by the javaagent)

```suggestion
include(":instrumentation:spring:spring-rmi-5.0:javaagent")
```

---

Got it, and thanks for the explanation. I didn't know of this convention.